src/binaryutils.cpp

```diff
@@ -265,6 +279,34 @@ InferenceEngine::Blob::Ptr convertMatsToBlob(std::vector<cv::Mat>& images, const
     }
 }
 
+Status convertBlobToMats(std::vector<cv::Mat>& images, const InferenceEngine::Blob::Ptr& blob, const std::shared_ptr<TensorInfo>& tensorInfo) {
+    int matPrecision = getMatTypeFromTensorPrecision(tensorInfo->getPrecision());
+    int rows, cols, channels;
+    if (tensorInfo->getLayout() == InferenceEngine::Layout::NCHW) {
```

Comment:
TensorInfo may contain 5 dimenions and layout ANY when demultiplexer is present in DAG. I'm afraid we are missing integration with DAGs here.